src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -767,7 +764,7 @@ private int getCountOfEmptyLinesBefore(int lineNo) {
         //  [lineNo - 2] is the number of the previous line
         //  because the numbering starts from zero.
         int lineBeforeIndex = lineNo - 2;
-        while (lineBeforeIndex >= 0
+        while (lineBeforeIndex > -1
```

Comment:
I need to recheck this change when I come to laptop, if it is not required for functional fix, I will request to move it to separate commit ( unrelated to issue) of this PR

Reply:
I'm investing this yet.

Reply:
We are slowly trying to implement more mutators, but its a long and tedious task. Some checks are so fragile that mutations start to bring forth ordering issues where we may not get the same result each run.
Passing current mutation just means we have reached the minimum required tests. It will never mean we have enough tests.

Reply:
yes, such changes are required to pass pitest, ok lets keep them.
It will be too hard to figure out test inputs for them.